exercises/palindrome-products/canonical-data.json

```diff
@@ -0,0 +1,126 @@
+{
+    "exercise": "palindrome-products",
+    "version": "1.0.0",
+    "cases": [{
+            "description": "finds the smallest palindrome from single digit factors",
+            "property": "smallest",
+            "input_min": 1,
+            "input_max": 9,
+            "expected": {
+                "value": 1,
+                "factors": [
+                    [1, 1]
+                ]
+            }
+        },
+        {
+            "description": "finds the largest palindrome from single digit factors",
+            "property": "largest",
+            "input_min": 1,
+            "input_max": 9,
+            "expected": {
+                "value": 9,
+                "factors": [
+                    [1, 9],
+                    [3, 3]
+                ]
+            }
+        },
+        {
+            "description": "find the largest palindrome from double digit factors",
+            "property": "largest",
+            "input_min": 10,
+            "input_max": 99,
```

Comment:
Missing test for the smallest in this range?
It is used as an example in the description, so I would expect to find it here.

Reply:
I added this test.